.gitignore

```diff
@@ -7,8 +7,6 @@ components/*/test/screenshots/golden/
 helpers/test/screenshots/current/
 helpers/test/screenshots/golden/
 lang/fr-on.js
-mixins/*/test/screenshots/current/
-mixins/*/test/screenshots/golden/
```

Comment:
`async-container-mixin` is the only `vdiff` test in `mixin` so we can clean this up now. You'll need to clean up any lingering  local copies next time you pull after this merges.